### DISCUSSION TITLE
Support for custom icon

### DISCUSSION THREAD

**Discussion by iddiek** [piitaya/lovelace-mushroom#1005] [Ideas]
There are many custom repo in hacs for different icon instead mdi:, ex. phu:
They works also in mushroom but no autofill and no preview is available. Is it a possible implement?

**Comment by piitaya** [comment]
Mushroom just use ms the ha icon picker. It's compatible with custom icons. I have the preview. I don't know why you don't have it.

![image](https://user-images.githubusercontent.com/5878303/216716754-7719961f-ac40-442f-ab85-a60a5530db33.jpeg)

**Reply by iddiek** [reply]
Not in template-card
